mav_msgs/include/mav_msgs/common.h

```diff
@@ -149,6 +149,84 @@ inline void skewMatrixFromVector(const Eigen::Vector3d& vec,
       0.0f;
 }
 
+inline void vectorFromSkewMatrix(const Eigen::Matrix3d& vec_skew,
+                                 Eigen::Vector3d* vec) {
+  assert(vec);
+  if ((vec_skew + vec_skew.transpose()).norm() < 1e-6){
+    *vec << vec_skew(2,1), vec_skew(0,2), vec_skew(1,0);
+  } else {
+    std::cerr << "[mav_msgs] Matrix is not skew-symmetric." << std::endl;
+    *vec = Eigen::Vector3d::Zero();
+    return;
```

Comment:
would it make sense to have this as a bool? so that it returns false if it's not skew symmetri